Sources/Sentry/include/SentryCrashIntegration.h

```diff
@@ -20,6 +20,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface SentryCrashIntegration : NSObject <SentryIntegrationProtocol>
 
+/**
+ * Wrapper for `SentryCrash.sharedInstance.systemInfo`, to cash the result.
+ *
+ * @return NSDictionary system info.
+ */
++ (NSDictionary *)systemInfo;
+
```

Comment:
I don't think we need to expose this publicly

Reply:
done